CMakeLists.txt

```diff
@@ -117,6 +117,24 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
 set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR})
 
+if (UNIX)
+  find_program(CCACHE_FOUND ccache)
+  if(CCACHE_FOUND)
```

Comment:
please add a flag that would explicitly enable ccache and log the error if ccache was not found

Reply:
so you want ccache to be used only when appropriate flag is passed to cmake?

In other words, do you want the default value of this flag to be OFF or ON?

Reply:
Yes: USE_CCACHE, which should be false by default.

Reply:
done